arangod/Cluster/ServerState.cpp

```diff
@@ -329,6 +333,32 @@ bool ServerState::unregister() {
   return r.successful();
 }
 
+ResultT<uint64_t> ServerState::readRebootIdFromAgency(AgencyComm& comm)
+{
+  TRI_ASSERT(!_id.empty());
+  std::string rebootIdPath = "Current/ServersKnown/" + _id + "/rebootId";
+  AgencyCommResult result = comm.getValues(rebootIdPath);
+
+  if (!result.successful()) {
+    LOG_TOPIC(WARN, Logger::CLUSTER)
+      << "Could not read back " << rebootIdPath;
+
+    ResultT<uint64_t>::error(TRI_ERROR_INTERNAL, "could not read rebootId from agency");
```

Comment:
```suggestion
    return ResultT<uint64_t>::error(TRI_ERROR_INTERNAL, "could not read rebootId from agency");
```

See also the fixes for `devel` and `3.5`